packages/backend/index.js

```diff
@@ -209,112 +328,3 @@ app.get("/users", async (req, res) => {
     res.status(500).json({ message: "Internal server error" });
   }
 });
-
-// Register a new user
-app.post("/users", async (req, res) => {
```

Comment:
Are you sure we can delete all these routes?

Reply:
This route wasn't deleted it was just moved to a different part of the file. The github diff makes it look confusing :/